packages/amplify-graphql-conversation-transformer/src/resolvers/assistant-response-stream-pipeline-definition.ts

```diff
@@ -0,0 +1,88 @@
+import { MappingTemplate } from '@aws-amplify/graphql-transformer-core';
+import { ConversationDirectiveConfiguration } from '../conversation-directive-configuration';
+import {
+  createResolverFunctionDefinition,
+  createS3AssetMappingTemplateGenerator,
+  PipelineDefinition,
+  ResolverFunctionDefinition,
+} from './resolver-function-definition';
+import { toUpper } from 'graphql-transformer-common';
+
+/**
+ * The pipeline definition for the assistant response stream mutation resolver.
+ */
+export const assistantResponseStreamPipelineDefinition: PipelineDefinition = {
+  requestSlots: [init(), auth(), verifySessionOwner()],
+  dataSlot: data(),
+  responseSlots: [],
+  field: (config) => ({ typeName: 'Mutation', fieldName: fieldName(config) }),
+};
+
+/**
+ * The init slot for the assistant response mutation resolver.
+ */
+function init(): ResolverFunctionDefinition {
```

Comment:
nit: We prefer arrow functions rather than `function` declarations. I'm not sure how this even [passed linting?](https://github.com/aws-amplify/amplify-category-api/blob/main/.eslintrc.js#L171C6-L171C41)